Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-create.js

```diff
@@ -48,27 +77,40 @@ $(document).ready(function () {
         numericTextBox.enable(true);
         input.attr("data-val-required", "Лимита е задължителен!");
 
-        $("form").removeData("validator");
-        $("form").removeData("unobtrusiveValidation");
-        $.validator.unobtrusive.parse($('form'));
+        $("#create-form form").removeData("validator");
+        $("#create-form form").removeData("unobtrusiveValidation");
+        $.validator.unobtrusive.parse($('#create-form form'));
     }
 
     $('#enable-sclimit').change(function () {
         if ($(this).is(':checked')) {
             numericTextBox.enable(true);
             input.attr("data-val-required", "Лимита е задължителен!");
-            
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+
+            $("#create-form form").removeData("validator");
+            $("#create-form form").removeData("unobtrusiveValidation");
+            $.validator.unobtrusive.parse($('#create-form form'));
         }
         else {
             numericTextBox.enable(false);
             input.removeAttr("data-val-required");
 
-            $("form").removeData("validator");
-            $("form").removeData("unobtrusiveValidation");
-            $.validator.unobtrusive.parse($('form'));
+            $("#create-form form").removeData("validator");
+            $("#create-form form").removeData("unobtrusiveValidation");
+            $.validator.unobtrusive.parse($('#create-form form'));
+        }
+    });
+
+    $("#create-form form").on("submit", function(ev) {
+        var submissionTypes = $('input[name^="SubmissionTypes"][name$=".IsChecked"]');
+        var submissionTypesValidation = $("span[data-valmsg-for='SelectedSubmissionTypes']");
+        if (submissionTypes.is(":checked")) {
+            submissionTypesValidation.attr('class', 'field-validation-valid');
+            submissionTypesValidation.text("");
+        } else {
+            submissionTypesValidation.attr('class', 'field-validation-error');
```

Comment:
Please be consistent with the type of quotes in JavaScript. Use only `'` or only `"`, but not both.